src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/getImageBlock.js

```diff
@@ -1,10 +1,10 @@
 const getImageBlock = (type, blockData, isAmp) => {
   const supportedImageTypes = ['idt2'];
 
-  if (!supportedImageTypes.includes(type)) return null;
-
   const imageData = blockData[type];
 
+  if (!supportedImageTypes.includes(type) || !imageData) return null;
```

Comment:
Should we add a test for this @rhenshaw56 ? Was this picked up in testing?

Reply:
> Should we add a test for this

Adding coverage for it now

> Was this picked up in testing?

Yes it was, some IDT2 on test don't have the amp image data cc @paruchurisilpa